README.md

```diff
@@ -110,8 +110,12 @@ There are 3 types of release:
     1. You can simply deploy to root of the `gh-pages` branch with: `npm run deploy`.
     2. You can clean (remove `br_` folders and clear root) your `gh-pages` branch and deploy to root in a single command with `npm run deploy:clean`
     3. You can deploy to a folder in your `gh-pages` branch in order to separate from root app deployment and other folder deployments with: `npm run deploy:folder br_test_folder` (folder name must be prefixed with `br_`)
-2. Release to staging: simply run `npm run deploy:staging` and follow the instructions.
-3. Release to production: simply run `npm run deploy:production` and follow the instructions.
+2. Release to staging: 
+    1. `git tag staging_v20191205 -m 'release staging'` # tag need to follow the format `staging.*`
```

Comment:
```suggestion
    1. `git tag staging_v20191205 -m 'release staging'` # the tag needs to follow the RegExp format `/^staging.*/`
```

Suggestion for clarity so reader can interpret as RegExp, instead of interpreting as a glob pattern. (Currently it reads like the prefix is `staging.` and not that `.` is a wildcard)